thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,37 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetImageDiff(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetImageDiff")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	opts1 := ImageDiffOptions{
```

Comment:
Like before, overwrite `opts` instead :
```suggestion
	opts := ImageDiffOptions{
```